Preload.pm

```diff
@@ -0,0 +1,56 @@
+=head1 LICENSE
+Copyright [2009-2016] EMBL-European Bioinformatics Institute
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=cut
+
+package EnsEMBL::REST::PreloadRegistry;
```

Comment:
I'm confused to the reason why this module exists. `EnsEMBL::REST::Model::Registry` has got a pre-load method in there which is run when the config for `<Model::Registry>` has `preload = 1` set.

From what I remember so long as you run the server in post-fork mode rather than pre-fork with `preload` set in the config Catalyst will do the rest.